timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -24,6 +26,8 @@ class SigmaPlugin(interface.BaseSketchAnalyzer):
     # this file.
     _RULES_PATH = ''
 
+    # A list of fields to include in the view output.
+    _FIELDS_TO_INCLUDE = ['message']
```

Comment:
You don't need this. Message is the default field if nothing is provided.

Reply:
ack, done